### ISSUE TITLE
Platform table - unnecessary 'Empty' announcement while using Screen Reader

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12787]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

1. Create table like in the [example](https://stackblitz.com/edit/sjxljz-b7nja9?file=src%2Fapp%2Ftable-custom-pagination-example.component.html). 
2. Use screen reader (tested with VoiceOver MacOs version: 14.7) and read a cell for action column.

It is announced as 'empty' because there is an invisible 'empty'  text inserted automatically. But in fact cell is not empty because we have a button with 'delete' action there. 
In this case 'empty' text should not be added because it can be misleading for blind users.

**Comment by mikerodonnell89** [comment]
In this case, I'd encourage the application developer to provide a value to the `ariaLabel` input of the fd-button with the trash icon. Naturally the screenreader will attempt to read the contents of the button, and because fundamental-ngx is flexible as to what the developer puts in the button, it should be up to the application developer to handle this
